src/app/app.py

```diff
@@ -1,18 +1,28 @@
 # src.py
 from src.repository import Repository
 from src.repository import RepositoryConfig
-from src.repository import RepositoryEnum
-from src.repository import RepositoryBigQueryEnum
+
+# from src.repository import RepositoryEnum
+# from src.repository import RepositoryBigQueryEnum
+# from src.repository import CloneConfig
+from src.repository import Clone
+from src.matric import CK
+from src.matric import MetricConfig
 
 # from src.project import generate_random_query
 from typing import NamedTuple
-from src.utils import Logger
+from src.utils import Logger, make_dir, remove_dir
+from pathlib import Path
 
 
 class AppConfig(NamedTuple):
     name: str
     repository: Repository
     repository_config: RepositoryConfig
+    clone: Clone
+    clone_config: list
+    metric_config: MetricConfig
+    metric: CK
```

Comment:
We need to clean this up.

Let's abstract this into these classes AppConfig, AppConfigDB, AppConfigGit with these properties
```
class AppConfig(NamedTuple):
     name: str
     metric_config: MetricConfig
     metric: CK

class AppConfigDB(AppConfig):
     repository: Repository
     repository_config: RepositoryConfig

class AppConfigGit(AppConfig):
     clone: Clone:
     clone_config: list
```